tests/basetest.rb

```diff
@@ -133,7 +133,6 @@ def setup
   def teardown
     @device.close unless @device.nil?
     @device = nil
-    GC.start
```

Comment:
http://stackoverflow.com/questions/6067139/ruby-garbage-collect

    The trouble is that calling this method is very expensive and can slow down your application considerably if used aggressively.